### PR TITLE
Scientific Calculator

### PR DESCRIPTION
**Pull request by dalampira**
Hello there! 
I have created an new scientific activity and added some basic buttons of possible functions for extended functionallity. I have also created a class that implements the factorial of a number so you can link it with the appropriate button when you feel like working on it in the future. 
Let me know what you think :)

### CODE REVIEW

File: app/src/debug/res/values/strings.xml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="app_launcher_name">Calculator_debug</string>
+    <string name="scientific_name">Scientific_debug</string>
```
Comment (tibbi) [inline app/src/debug/res/values/strings.xml]: remove this line, just use default app name in calculator mode too

File: app/src/main/kotlin/com/simplemobiletools/calculator/activities/ScientificActivity.kt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+class ScientificActivity : SimpleActivity() {
+
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.scientific_activity)
+        appLaunched(BuildConfig.APPLICATION_ID)
+        
+        updateViewColors(calculator_holder, config.textColor)
+
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/calculator/activities/ScientificActivity.kt]: fix empty lines here and everywhere else too, just like adding spaces before brackets etc

File: app/src/main/kotlin/com/simplemobiletools/calculator/operation/FactorialOperation.kt [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+class ParOperation(value: Double) : UnaryOperation(value), Operation {
+
+    override fun getResult(): Double {
+        var result = 1.0
+        if (value==0.0 || value==1.0){
+            return result
+        } else {
+            var base: Int = value.toInt()
+            for (i in 1..base){
+                //Find Factorial of a number using for loop
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/calculator/operation/FactorialOperation.kt]: remove this comment, fix spaces, remove type declaration from "base", it isnt needed in such cases

**Comment by tibbi** [comment]
okay, it looks pretty good so far, implementing the functionality will be harder though :) Thanks

**Comment by dalampira** [comment]
Thanks for merging! I will take a look on implementing the functionality and let you know in time! :)

**Pull request merged**